src/main/java/seedu/ddd/logic/commands/ListCommand.java

```diff
@@ -53,46 +44,9 @@ public class ListCommand extends Command {
             + PREFIX_DESC + "wedding";
 
     public static final String MESSAGE_USAGE = COMMAND_DESCRIPTION + "\n"
-        + COMMAND_USAGE + "\n"
-        + CLIENT_EXAMPLE_USAGE + "\n"
-        + VENDOR_EXAMPLE_USAGE + "\n"
-        + EVENT_EXAMPLE_USAGE + "\n";
-
-    public static final String MESSAGE_SUCCESS = "Listed all contacts";
-
-    private static final Predicate<Event> CLEAR_EVENTS = any -> false;
-
-    private final Predicate<Contact> predicate;
-
-    public ListCommand(Predicate<Contact> predicate) {
-        this.predicate = predicate;
-    }
-
-    @Override
-    public CommandResult execute(Model model) {
-        requireNonNull(model);
-        model.updateFilteredEventList(CLEAR_EVENTS);
-        model.updateFilteredContactList(predicate);
-        return new CommandResult(String.format(MESSAGE_CONTACTS_LISTED_OVERVIEW,
-                model.getFilteredContactList().size()));
-    }
-
-    @Override
-    public boolean equals(Object other) {
-        if (other == this) {
-            return true;
-        }
-        // instanceof handles nulls
-        if (!(other instanceof ListCommand)) {
-            return false;
-        }
-        ListCommand otherListCommand = (ListCommand) other;
-        return predicate.equals(otherListCommand.predicate);
-    }
-    @Override
-    public String toString() {
-        return new ToStringBuilder(this)
-                .add("predicate", predicate)
-                .toString();
-    }
+            + COMMAND_USAGE + "\n"
+            + CLIENT_EXAMPLE_USAGE + "\n"
+            + VENDOR_EXAMPLE_USAGE + "\n"
+            + EVENT_EXAMPLE_USAGE + "\n";
+    public abstract CommandResult execute(Model model);
```

Comment:
The abstract execute model is not needed as it is specified in the parent Command class and must be overridden in the subclasses.

Reply:
Yep, I can remove it.